### PR TITLE
Compute invoice items outside AdditionalItemsResult.getAdditionalItems

### PR DESCRIPTION
**Pull request by reshmabidikar**
Fix to compute invoice items outside of `AdditionalItemsResult.getAdditionalItems` as per [this comment](https://github.com/killbill/killbill-avatax-plugin/pull/30#issuecomment-1357284521). With this fix, no changes would be required on the Kill Bill side.

### CODE REVIEW

File: src/main/java/org/killbill/billing/plugin/avatax/api/AvaTaxInvoicePluginApi.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*
- * Copyright 2014-2020 Groupon, Inc
- * Copyright 2020-2021 Equinix, Inc
- * Copyright 2014-2021 The Billing Project, LLC
+ * Copyright 2014-2022 Groupon, Inc
```
Comment (pierre) [inline src/main/java/org/killbill/billing/plugin/avatax/api/AvaTaxInvoicePluginApi.java:2]: 2020 :-)

Rest looks good.

File: src/main/java/org/killbill/billing/plugin/avatax/api/TaxRatesInvoicePluginApi.java [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /*
- * Copyright 2014-2020 Groupon, Inc
- * Copyright 2020-2021 Equinix, Inc
- * Copyright 2014-2021 The Billing Project, LLC
+ * Copyright 2014-2022 Groupon, Inc
```
Comment (pierre) [inline src/main/java/org/killbill/billing/plugin/avatax/api/TaxRatesInvoicePluginApi.java:2]: 2020

**Review by pierre — APPROVED** [review]

File: src/main/java/org/killbill/billing/plugin/avatax/api/AvaTaxInvoicePluginApi.java (hunk 1 above)
Comment (reshmabidikar) [reply src/main/java/org/killbill/billing/plugin/avatax/api/AvaTaxInvoicePluginApi.java]: Fixed

File: src/main/java/org/killbill/billing/plugin/avatax/api/TaxRatesInvoicePluginApi.java (hunk 2 above)
Comment (reshmabidikar) [reply src/main/java/org/killbill/billing/plugin/avatax/api/TaxRatesInvoicePluginApi.java]: Fixed

**Pull request merged**